dom/events/scrolling/wheel-event-transactions-target-display-change.tentative.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1">
+<meta name="variant" content="?include=none"/>
+<meta name="variant" content="?include=contents"/>
```

Comment:
Yeah, tricky case. So, if this could have `.relatedTarget` checks, it'd be nicer.

Reply:
See github.com/web-platform-tests/wpt/pull/37445/files#r1414207219